packages/flutter/lib/src/services/text_input.dart

```diff
@@ -2562,56 +2568,69 @@ class SystemContextMenuController with SystemContextMenuClient {
 
   /// Shows the system context menu anchored on the given [Rect].
   ///
+  /// Currently only supported on iOS 16.0 and later. Check
+  /// [MediaQuery.maybeSupportsShowingSystemContextMenu] before calling this.
+  ///
   /// The [Rect] represents what the context menu is pointing to. For example,
   /// for some text selection, this would be the selection [Rect].
   ///
+  /// `items` specifies the buttons that appear in the menu. Any `items` that
+  /// have no title will be passed a default title from the given
```

Comment:
Is the note about `localizations` necessary here since `ItemData` adheres to the platform API and the object itself does not do localizations.

Reply:
No good catch. That's leftover from previous iterations.